rosbag2_bag_v2_plugins/src/rosbag2_bag_v2_plugins/storage/rosbag_v2_storage.cpp

```diff
@@ -30,6 +30,11 @@
 namespace rosbag2_bag_v2_plugins
 {
 
+namespace
+{
+const char * IDENTIFIER = "rosbag_v2";
```

Comment:
```suggestion
constexpr const char * const IDENTIFIER = "rosbag_v2";
```

the ns is ok, as we discussed https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnonexported-entities